modules/doc/content/application_development/profiling.md

```diff
@@ -7,6 +7,66 @@ installation process.  Although it works best on Linux platforms, it also
 works reasonably well on Mac OS.  Instruments also works well for profiling
 applications on Mac systems.
 
+## Install Google's gperftools
+
+If your environment package from the MOOSE installation process has included Google's gperftools,
+you can skip this step.
+To install Google's gperftools on your own, our recommended procedure is to
+first install `libunwind`:
+
+```
+cd $HOME
+git clone git@github.com:libunwind/libunwind.git
+cd libunwind
+autoreconf -i
+./configure --prefix=$PWD/installed
+make -j$MOOSE_JOBS
+make install
+```
+
+and then the `gperftools` library:
+
+```
+cd $HOME
+git clone git@github.com:gperftools/gperftools.git
+cd gperftools
+./autogen.sh
+CPPFLAGS=-I$HOME/libunwind/installed/include \
+LDFLAGS="-L$HOME/libunwind/installed/lib -Wl,-rpath,$HOME/libunwind/installed/lib" \
```

Comment:
Is it better to let `configure` to take `libunwind_dir`? Instead of these flags

Reply:
`configure` will not find `libunwind` if it's not in standard system paths unless you pass these flags

Reply:
I do not think that gperftools configure accepts something like a `LIBUNWIND_DIR`:
```
(base) lindad@pop-os:~/gperftools(master)$ ./configure --help | grep unwind
  --enable-libunwind      enable libunwind linking
(base) lindad@pop-os:~/gperftools(master)$ ./configure --help | grep UNWIND
```

Reply:
I was surprised. Google should hire us to implement "--download-libunwind" and "--download-pprof" .

That will make users life a bit easier